docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Reduce database size using Bonsai Tries
+sidebar_position: 12
+description: Enable this feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To decrease the database size using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage policy in Besu, enable the early access feature `--Xbonsai-limit-trie-logs-enabled`. 
+When enabled, this feature can reduce database growth by more than 3GB each week on mainnet.
+
+## Limit Trie Logs for Bonsai
+
+:::caution
+
+The following commands are examples. Before executing these example commands on your node, modify them to apply to your node's configuration.
+
+:::
+
+1. Add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](configuration-file).
+1. Stop Besu. If Besu is running in a terminal, you can usually stop it by pressing `Ctrl+C`.
+1. (optional) Run the Besu client. Specify the Bonsai Trie data storage format and a data directory for the pruned trie logs:
+
+    ```bash title="Specify Bonsai Trie for pruned logs"
```

Comment:
```suggestion
    ```bash title="Specify Bonsai Trie for pruned logs"
```

Reply:
Think we should consistently use either "Trie Logs" or "trie logs".